### PR TITLE
follow_joint_trajectory_controller_handle: publish new multi_dof_trajectory field

### PR DESCRIPTION
**Pull request by JafarAbdi**
### Description

Use the new [multi_dof_trajectory](https://github.com/ros-controls/control_msgs/blob/galactic-devel/control_msgs/action/FollowJointTrajectory.action#L4) field, this have to be merged after galactic support

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/ros-planning/moveit/blob/master/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://ros-planning.github.io/moveit_tutorials/doc/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Review by henningkayser — APPROVED** [review]
Changes look reasonable to me. It's a little odd that MoveIt would complain about multi-dof trajectories even if this should be up to the controller implementation. Do we actually have a controller that supports this? In the future we should probably remove or at least simplify the controller plugins now that we don't use the fake controller anymore.

**Review by DLu — APPROVED** [review]

File: moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp [hunk 1]
```diff
@@ -48,18 +48,14 @@ bool FollowJointTrajectoryControllerHandle::sendTrajectory(const moveit_msgs::ms
...
-  }
-
   if (done_)
     RCLCPP_INFO_STREAM(LOGGER, "sending trajectory to " << name_);
   else
     RCLCPP_INFO_STREAM(LOGGER, "sending continuation for the currently executed trajectory to " << name_);
 
   control_msgs::action::FollowJointTrajectory::Goal goal = goal_template_;
   goal.trajectory = trajectory.joint_trajectory;
+  goal.multi_dof_trajectory = trajectory.multi_dof_joint_trajectory;
```
Comment (henningkayser) [inline moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp:58]: so... this doesn't seem to exist?

Comment (JafarAbdi) [reply moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp:58]: This only exists in galactic

Comment (henningkayser) [reply moveit_plugins/moveit_simple_controller_manager/src/follow_joint_trajectory_controller_handle.cpp:58]: ok, so we need to hold this for now until we branch off Foxy

**Pull request merged**